Source/Plugins/Core/com.equella.core/src/com/tle/web/wizard/section/RootWizardSection.java

```diff
@@ -305,6 +305,11 @@ void validateMandatoryFields(SectionInfo info, WizardState state) {
     PagesSection ps = info.lookupSection(PagesSection.class);
     wizardService
         .getWizardPages(state)
-        .forEach(p -> wizardService.ensureInitialisedPage(info, p, ps.getReloadFunction(), true));
+        .forEach(
+            p -> {
+              if (p.isViewable()) {
+                wizardService.ensureInitialisedPage(info, p, ps.getReloadFunction(), true);
+              }
+            });
```

Comment:
So as to stay within the functional paradigm a bit, how's about:

```suggestion
        .stream().filter(WebWizardPage::isViewable)
        .forEach(p -> wizardService.ensureInitialisedPage(info, p, ps.getReloadFunction(), true));
```

I think this also has a slight readability benefit.